onnxruntime/core/providers/dnnl/subgraph/dnnl_matmul.cc

```diff
@@ -139,14 +158,14 @@ void DnnlMatMul::CreatePrimitive(DnnlSubgraphPrimitive& sp, DnnlNode& node) {
   if (transA || transBatchA) {
     src_md = transposedA_md;
   } else {
-    src_md = dnnl::memory::desc(src_dims, node.Input(IN_A).Type(), dnnl::memory::format_tag::any);
+    src_md = dnnl::memory::desc(src_dims, node.Input(IN_A).Type(),  get_default_format(src_dims));
```

Comment:
please run `lintrunner -a`. there is extra space after `,`

Reply:
thanks, i have pushed the update and checking how it was missed when i ran lintrunner -a on this version.